### PR TITLE
OSDOCS#8127:LSO docs should emphasize that multiple LV sets are not s…

### PR DESCRIPTION
**Pull request by lpettyjo**
…upported

Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OSDOCS-8127

Link to docs preview: https://67046--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-local#local-storage-discovery_persistent-storage-local

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @RomanBednar

### CODE REVIEW

**Comment by skrthomas** [comment]
@lpettyjo can you add a link to the preview? TY!!

File: modules/persistent-storage-local-discovery.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ Use the following procedure to automatically discover local devices, and to auto
 
 [WARNING]
 ====
-Use the `LocalVolumeSet` object with caution. When you automatically provision persistent volumes (PVs) from local disks, the local PVs might claim all devices that match. If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node.
+Use the `LocalVolumeSet` object with caution. When you automatically provision persistent volumes (PVs) from local disks, the local PVs might claim all devices that match. If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node. Ceating multiple instances of `LocalVolumeSet` that target a node more than once is not supported.
```
Comment (skrthomas) [inline modules/persistent-storage-local-discovery.adoc:24]:
```suggestion
Use the `LocalVolumeSet` object with caution. When you automatically provision persistent volumes (PVs) from local disks, the local PVs might claim all devices that match. If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node. Creating multiple instances of a `LocalVolumeSet` that target a node more than once is not supported.
```

**Review by skrthomas — COMMENTED** [review]
A couple of small nits. /lgtm otherwise!

File: modules/persistent-storage-local-discovery.adoc (hunk 1 above)
Comment (skrthomas) [reply modules/persistent-storage-local-discovery.adoc:24]: I also wonder about rewording this sentence:
There is no support for creating multiple instances of a `LocalVolumeSet` that target a node more than once.

**Comment by lpettyjo** [comment]
> @lpettyjo can you add a link to the preview? TY!!

Done. https://67046--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-local#local-storage-discovery_persistent-storage-local

**Pull request merged**